### PR TITLE
Fix a bug and improve here and there

### PR DESCRIPTION
**Pull request by Offirmo**
Hello @hrishikeshs !

I tried your latest version and unfortunately it failed
1. because of the missing jsdom dependency --> so I improved the readme to better explain that
2. because of a bug in calling jsdom with wrong parameters --> fixed it

I also made improvements here and there :
- added an editorConfig so that my editor correctly picks you indentation style
- running the tests, I saw you used an intermediate makefile which I removed : package.json can do it better
- looking at the tests, I reorg'd them to be real BDD tests :)

Enjoy !

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -1,8 +1,10 @@
 (function() {
   if(typeof document === "undefined") {
+    // node.js
     var jsdom = require('jsdom');
-    document = jsdom.jsdom("<html><body><h1>Hello</h1></body></html>", null, {
+    document = jsdom.jsdom("<html><body><h1>Hello</h1></body></html>", {
```
Comment (Offirmo) [inline index.js]: here the extra "null" was breaking.

File: package.json [hunk 2]
```diff
@@ -4,7 +4,7 @@
   "description": "Utility that converts your JSON to a html representation",
   "main": "index.js",
   "scripts": {
-    "test": "make test"
+    "test": "mocha --reporter nyan"
```
Comment (Offirmo) [inline package.json]: simpler, isn't it ?

**Comment by hrishikeshs** [comment]
Thanks a lot for doing this! 👍👍 😄 
I'm outside currently, will merge once I get home.

**Comment by Offirmo** [comment]
My pleasure.

**Comment by Offirmo** [comment]
Be careful : your latest tag (0.1.3) doesn't match package.json version (0.1.1)

What you should do :
1. fix package.json to 0.1.3, commit
2. execute this :
```bash
npm install -g mversion
mversion patch -m
git push
git push --tags
npm publish
```

and all will be clear !

**Comment by hrishikeshs** [comment]
@Offirmo, all done and updated! Thanks for the time you took to help me out! you're awesome! :smile:

**Pull request merged**